src/hooks/useDebouncedSearch.ts

```diff
@@ -104,13 +106,16 @@ export const useDebouncedSearch = <
       return;
     }
     try {
+      setIsSearching(true);
       const response = await index.search<Response>(query, params);
       trackGoal(FATHOM_SEARCH_PERFORMED_EVT_ID, 0);
       setResults(transformResponse(response));
+      setIsSearching(false);
     } catch (e) {
       console.error(`Search for query "${query}" failed (${e})`);
+      setIsSearching(false);
```

Comment:
Can move this and the one on 113 outside the `catch` to only call it once!

Reply:
32a4b010b7a3d5b76bec8ff5af8e63db634fead5

Reply:
ty ty, pre-stamped btw.